src/hyp3_autorift/process.py

```diff
@@ -115,8 +115,11 @@ def get_s2_manifest(scene_name):
 def get_s2_path(scene_name: str) -> str:
     bucket = 'its-live-project'
     key = f's2-cache/{scene_name}_B08.jp2'
-    if s3_object_is_accessible(bucket, key):
-        return f'/vsis3/{bucket}/{key}'
+    try:
+        if s3_object_is_accessible(bucket, key):
+            return f'/vsis3/{bucket}/{key}'
+    except botocore.exceptions.NoCredentialsError:
+        log.info('No credentials found')
```

Comment:
Ah, yes, this assumes you've set up AWS credentials in some form. I think instead of here, I'd move that except inside `s3_object_is_accessbile`:
https://github.com/ASFHyP3/hyp3-autorift/blob/develop/src/hyp3_autorift/process.py#L169-L176

Reply:
So it'd look something like:
```python
def s3_object_is_accessible(bucket, key):
    try:
        S3_CLIENT.head_object(Bucket=bucket, Key=key)
    except botocore.exceptions.ClientError as e:
        if e.response['Error']['Code'] in ['403', '404']:
            return False
        raise
    except botocore.exceptions.NoCredentialsError:
        log.info('No AWS credentials found')  # Maybe a warning instead?
        return False
    return True